src/main/kotlin/no/nav/syfo/fastlege/ws/adresseregister/AdresseregisterClient.kt

```diff
@@ -35,6 +35,43 @@ class AdresseregisterClient(
             throw e
         }
 
+    fun hentBehandlereForKontor(parentHerId: Int) =
+        try {
+            val wsOrg = adresseregisterSoapClient.getOrganizationDetails(parentHerId)
+            wsOrg.people.organizationPersons.map { orgPerson ->
+                Behandler(
+                    aktiv = orgPerson.isActive,
+                    herId = orgPerson.herId,
+                    personIdent = orgPerson.person.citizenId.id,
+                    hprId = orgPerson.person.hprInformation.hprNumber,
+                    fornavn = orgPerson.person.firstName,
+                    mellomnavn = orgPerson.person.middleName,
+                    etternavn = orgPerson.person.lastName,
+                    kategori = orgPerson.person.hprInformation.authorizations?.authorizations?.map { aut ->
```

Comment:
Strevde lenge med å finne ut hvordan man skulle gjøre dette. Men ser ut som det funker nå - får LE for leger og andre verdier som stemmer overens med kodeverket BehandlerKategori i isdialogmelding.

Reply:
<img src="https://media1.giphy.com/media/iI3kpnBS3q7u7STTWG/giphy.gif"/>